### ISSUE TITLE
ImportError bug in Release 0.2.4

### ISSUE THREAD

**Issue by marcosdiezgarcia** [Fujitsu-UTokyo-QDD/QDD#68]
I am using Python 3.10.0 and installed QDD (qdd-0.2.4) with default options on a fresh Python virtual environment by running: `pip install qdd`.

However, running `import qdd` from Python REPL unexpectedly raises an `ImportError` exception. The stacktrace is:

```python
>>> import qdd
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/marcos/qar/QDD/qdd/__init__.py", line 3, in <module>
    from .qdd_backend import QddBackend
  File "/home/marcos/qar/QDD/qdd/qdd_backend.py", line 26, in <module>
    from qdd import pyQDD
ImportError: cannot import name 'pyQDD' from partially initialized module 'qdd' (most likely due to a circular import) (/home/marcos/qar/QDD/qdd/__init__.py)

```

I obtain the same error also in Python 3.11.0 and 3.12.5.
Installing QDD via `pip install qdd` seems to have some issue.

**Comment by kkyusuke** [comment]
Thanks for reporting the issue.
I feel that there are two causes, but not sure. Please help us to solve this problem.

### 1. Compatibility problem?
It may be the compatibility issue, so could you share your environment? (e.g., Ubuntu 22.04 x86_64 or RHEL8 arm64)

### 2. Conflict?
It seems you have downloaded QDD manually to `/home/marcos/qar/QDD`. If you are in that directory, pip's QDD and the manually-downloaded QDD may conflict each other.
Could you run python under an empty directory?

**Comment by marcosdiezgarcia** [comment]
Update:

1. I was using a machine with Ubuntu 20.04 LTS. I think there is no compatibility problem.
2. I tried running the example Python code `import qdd` from a different directory and saw no issue. So indeed the issue seems to be that the git-cloned version shadows the pip-installed version.

I tried reinstalling qdd (develop) using Python 3.12.5. The installation works but only if done from source using cmake. However, I needed to run `python -m pip install .` (the installation instructions in README do not mention this step after `python -m build`).
